### PR TITLE
WIP Added Admin Panel

### PR DESCRIPTION
**Pull request by osusara**
- Added a link to Admin Panel in the navbar dropdown (this link is available only for users with admin permissions).
- Changed Admin Panel layout. Included (add users to product, add product, add organization).
- Added functionality to add multiple users to products.
- Done necessary validations.
- Changes in package.json files, buildspec.yml, and README.md.

### CODE REVIEW

File: web/src/app/shared/components/nav-bar/nav-bar.component.ts [hunk 1]
```diff
@@ -8,19 +8,24 @@ import { AuthService } from '@shared/services/auth/auth.service';
...
+  }
 
   ngOnInit() {
   }
+
   async getUser() {
     this.currentUser = await this.authService.getCurrentUser();
     if ( this.currentUser ) {
       this.userName = this.currentUser.given_name + ' ' + this.currentUser.family_name;
+      this.isAdmin = this.currentUser.admin === '1' ? true : false;
```
Comment (CharithW) [inline web/src/app/shared/components/nav-bar/nav-bar.component.ts]: Ternary operation can be removed from here

Comment (osusara) [reply web/src/app/shared/components/nav-bar/nav-bar.component.ts]: I'll correct it

File: web/src/app/admin/admin.api.service.ts [hunk 2]
```diff
@@ -12,7 +12,9 @@ export class AdminApiService {
   constructor(private httpClient: HttpClient) {}
 
   public async getAllProducts(): Promise<Product[]> {
-    const result = await this.httpClient.get(adminRoute + '/products').toPromise();
+    const result = await this.httpClient
+      .get(adminRoute + '/products')
```
Comment (CharithW) [inline web/src/app/admin/admin.api.service.ts]: use string interpolation.
`${adminRoute}/products`

Comment (osusara) [reply web/src/app/admin/admin.api.service.ts]: Working on it

**Pull request merged**